testing/src/main/java/io/stargate/it/http/GraphqlTest.java

```diff
@@ -829,6 +826,48 @@ public void insertProducts() {
     assertThat(product.getDescription()).hasValue(input.description());
   }
 
+  @Test
+  public void bulkInsertProducts() {
+    ApolloClient client = getApolloClient("/graphql/betterbotz");
+
+    String productId1 = UUID.randomUUID().toString();
+    String productId2 = UUID.randomUUID().toString();
+    ProductsInput product1 =
+        ProductsInput.builder()
+            .id(productId1)
+            .name("Shiny Legs")
+            .price("3199.99")
+            .created(now())
+            .description("Normal legs but shiny.")
+            .build();
+    ProductsInput product2 =
+        ProductsInput.builder()
+            .id(productId2)
+            .name("Non-Legs")
+            .price("1000.99")
+            .created(now())
+            .description("Non-legs.")
+            .build();
+
+    bulkInsertProducts(client, Arrays.asList(product1, product2));
```

Comment:
verify result of the insert as well..

Reply:
There is a verification hidden in the `bulkInsertProducts` method.